non-critical-infra/hosts/caliban/nixpkgs-swh.nix

```diff
@@ -0,0 +1,32 @@
+{ inputs, config, ... }:
+let
+  cfg = config;
+in
+{
+  imports = [
+    inputs.nixpkgs-swh.nixosModules.nixpkgs-swh
+  ];
+  config = {
```

Comment:
Why is this nested into the config attrset? That looks redundant.

Reply:
It is and addressed by https://github.com/NixOS/infra/pull/834